SignalServiceKit/src/Util/OWSSwiftUtils.swift

```diff
@@ -20,3 +20,16 @@ public func owsFail(_ message: String) {
     Logger.flush()
     assertionFailure(message)
 }
+
+// Once we're on Swift4.2 we can mark this as inlineable
+// @inlinable
+@objc
```

Comment:
@michaelkirk I'm getting a build error doing something similar in the "prod fail" methods.

> @objc can only be used with members of classes, @objc protocols, and concrete extensions of classes